Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -5556,6 +5556,64 @@ RefPtr<CSSValue> consumeSize(CSSParserTokenRange& range, CSSParserMode mode)
     return CSSValueList::createSpaceSeparated(pageSize.releaseNonNull(), orientation.releaseNonNull());
 }
 
+RefPtr<CSSValue> consumeTextTransform(CSSParserTokenRange& range)
+{
+    if (consumeIdentRaw<CSSValueNone>(range))
+        return CSSValueList::createSpaceSeparated(CSSPrimitiveValue::create(CSSValueNone));
```

Comment:
You could return `CSSPrimitiveValue::create(CSSValueNone)` directly without putting it in a list.